index.html

```diff
@@ -21,86 +21,243 @@
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
                     </ul>
                 </div>
 
                 <div class="column column2">
                     <ul class="styling">
                         <li class="title">Frameworks</li>
-                       <li>Bootstrap</li>
-                       <li>Ruby on Rails</li>
-                       <li>RSPec</li>
-                       <li>CapyBara</li>
-                       <li>Selenium</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
                     </ul>
                 </div>
 
                 <div class="column column3">
-                   
+
                     <ul class="styling">
                         <li class="title">Skills</li>
-                       <li>Codekit</li>
-                       <li>Github</li>
-                       <li>Codepen</li>
-                       <li>Gitlab</li>
-                       <li>Terminal</li>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
                     </ul>
                 </div>
-               
 
-               </div>
-               
-                <img src="Vector.4.svg" alt="separator">
-               
 
-               <div class="icon">
-                    
+            </div>
+
+            <img src="Vector.4.svg" alt="separator" class="separator-text">
+
+
+            <div class="icon">
+
                 <ul class="styling1">
                     <li class="styling1"><img src="Vector.1.svg" alt="icon"></li>
                     <li class="styling1"><img src="Group.png" alt="icon"></li>
-                   <li class="styling1"><img src="Vector.3.svg" alt="icon"></li>
-                   <li class="styling1"><img src="Vector.icon.svg" alt="icon"></li>
-                   <li class="styling1"><img src="Vector.5.svg" alt="icon"></li>
-                   
+                    <li class="styling1"><img src="Vector.3.svg" alt="icon"></li>
+                    <li class="styling1"><img src="Vector.icon.svg" alt="icon"></li>
+                    <li class="styling1"><img src="Vector.5.svg" alt="icon"></li>
+
                 </ul>
             </div>
 
             <div class="footer"></div>
             <ul class="styling2">
                 <li class="styling2"><img src="Group.55.svg" alt="icon"></li>
-                <li class="styling2"><p>Work</p></li>
-              
-               
+                <li class="styling2">
+                    <p>Work</p>
+                </li>
+
+
             </ul>
-        
+</section>
+            <section>
```

Comment:
- [ ] Please ensure that the padding/margin around this section matches that of the Figma
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/170705211-a340d3fe-6a8e-452e-9ede-9d11aa9d18a9.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/170705290-43bdea36-c27d-4d91-a477-6fa14d45d6be.png)